source/docs/software/advanced-gradlerio/compiler-args.rst

```diff
@@ -0,0 +1,39 @@
+Using Compiler Arguments
+============================
+
+Compiler arguments allow us to change the behavior of our compiler. This includes making warnings into errors, ignoring certain warnings and choosing optimization level. Normally we would pass them in as flags when compiling our code but this doesn't work in GradleRIO. Instead we have to modify our build.gradle.
```

Comment:
I would add a link to where the default flags are located

Reply:
where would one find the default flags?

Reply:
https://github.com/wpilibsuite/native-utils/blob/0055bc77da9d9d37e0876d04af53b7317b78174f/src/main/java/edu/wpi/first/nativeutils/WPINativeUtilsExtension.java#L37

Reply:
There are a fair few there so I am just adding a link to them for readibility